src/WebJobs.Script/Description/DotNet/DiagnosticExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.WebJobs.Script.Description.DotNet.CSharp.Analyzers
+{
+    internal static class DiagnosticExtensions
+    {
+        public static object[] GetDiagnosticMessageArguments(this Diagnostic diagnostic)
+        {
+            return typeof(Diagnostic)
```

Comment:
Unfortunately, not. I had a workaround that pulled from the formatted message but to support multiple arguments it was getting a bit cumbersome. The included test would fail and let us know if this breaks down with future updates, but I have opened an issue here so see if the Roslyn team can make this public: https://github.com/dotnet/roslyn/issues/11523